tsconfig.json

```diff
@@ -1,16 +1,22 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "esModuleInterop": true,
     "module": "commonjs",
-    "noImplicitAny": true,
-    "removeComments": true,
-    "preserveConstEnums": true,
+    "noErrorTruncation": false,
     "outDir": "bin",
-    "sourceMap": true
+    "preserveConstEnums": true,
+    "removeComments": true,
+    "sourceMap": true,
+    "strict": true,
+    "strictNullChecks": false,
+    "target": "es2018",
```

Comment:
This assumes things like await and spread syntax.  Should be fine for node 10.X

Does talon have an official list of "supported node versions"?

Reply:
@cagerton It does not, but we could add it to the `package.json`.